commercetools-models/src/main/java/io/sphere/sdk/orders/OrderFromCartDraft.java

```diff
@@ -36,11 +52,32 @@
     @Nullable
     ShipmentState getShipmentState();
 
+    static OrderFromCartDraft of(final Cart cart, @Nullable final String orderNumber, @Nullable final PaymentState paymentState) {
+        ResourceIdentifier<Cart> cartResourceIdentifier = ResourceIdentifier.ofId(cart.getId());
+
+        return OrderFromCartDraftDsl.of(cartResourceIdentifier, cart.getVersion(), orderNumber, paymentState);
+    }
+
+    /**
+     * getId has been @deprecated so this method has been replaced with {@link}
+     */
+    @Deprecated
     static OrderFromCartDraft of(final Versioned<Cart> cart, @Nullable final String orderNumber, @Nullable final PaymentState paymentState) {
         return OrderFromCartDraftDsl.of(cart.getId(), cart.getVersion(), orderNumber, paymentState);
     }
 
+    static OrderFromCartDraft of(final Cart cart) {
+        ResourceIdentifier<Cart> cartResourceIdentifier = ResourceIdentifier.ofId(cart.getId());
+
+        return OrderFromCartDraftDsl.of(cartResourceIdentifier, cart.getVersion());
+    }
+    
+    /**
+     * getId has been @deprecated so this method has been replaced
+     */
+    @Deprecated
```

Comment:
This method can be removed as a Cart extends Versioned and instead refactor the other of method to create and use a resource identifier

Reply:
replacing with static `OrderFromCartDraft of(final ResourceIdentifier<Cart> cartResourceIdentifier, final Long version)`